Gruntfile.js

```diff
@@ -122,6 +108,18 @@ module.exports = function (grunt) {
           browsers
         }
       },
+      compatibilityWithWebpackProd: {
```

Comment:
I duplicated the **compatibilityWithWebpack** instead of reusing it to be more explicit about its meaning. 

Just in case in the future we want to make them do different stuff

Reply:
I'm a little torn about this one. The other compat tests use karma to do the bundling, and I was sort of imagining this compat test to do the same. Is there a reason why can't configure the "webpack" preprocessor being used by karma here to run in production mode?